src/App.jsx

```diff
@@ -14,70 +16,64 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGoods, setSelectedGoods] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const handleSelectGoods = (good = '') => {
+    const toSelect = good === selectedGoods ? '' : good;
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+    setSelectedGoods(toSelect);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGoods.length
```

Comment:
Why you check length if selectedGood this is the string and this variable has only two states, some value when good selected and empty string when not selected? I think it will be enough
```suggestion
        {selectedGoods
```